src/firefly/js/templates/lightcurve/LcPeriodogram.jsx

```diff
@@ -424,6 +430,26 @@ function stepsizeValidator(description) {
                       {valid: false, message: description + `: must be greater than ${min}`};
     };
 }
+/**
+ * @summary peaks number validator
+ * @param description
+ * @returns {Function}
+ */
+function peaksValidator(description) {
+    return (valStr) => {
+        if (!valStr) return {valid: true};
+        var retval = Validate.isInt(description, valStr);
+
+        if (!retval.valid) return retval;
+
+        var val = parseInt(valStr);
+        const min = 0;
+        var bVal = val > min;
+
+        return bVal ? {valid: true} :
+        {valid: false, message: description + `: must be greater than ${min}`};
```

Comment:
API is not ready, default values are not communicated. For now, is open to the user to know what they enter. Once we get better understanding of the API, we will add validation to input fields.